sp_peaks/slics.py

```diff
@@ -69,3 +69,116 @@ def read_catalogue(file_path, all_col=True):
     dat = ascii.read(file_path, names=col_names, include_names=include_names)
 
     return dat
+
+import numpy as np
+
+def read_SLICS_cats(file_paths):
```

Comment:
it looks like this is already covered by read_catalog

Reply:
Not quite. The `read_catalogue` function reads a catalogue and returns its content (columns of ra, dec, gamma, etc.), while `read_SLICS_cats` reads the information from filenames of the catalogues (id, bin, seed, LOS and tile) and returns a recarray with those. Probably the naming of `read_SLICS_cats` is confusing, I will change it to something better.